cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -240,6 +243,16 @@ public void init(@Nullable final NotificationOpenedListener notificationOpenedLi
         init(channelId, notificationOpenedListener);
     }
 
+    /**
+     * initialize CleverPush SDK with notification opened callback
+     *
+     * @param notificationOpenedCallbackListener callback for the notification opened
+     */
+    public void init(@Nullable final NotificationOpenedCallbackListener notificationOpenedCallbackListener) {
```

Comment:
We di not want to add a new init method. It should work with existing init methods (similar to ReceivedCallbackListener)

Reply:
okay I removed